mealie/schema/_mealie/datetime_parse.py

```diff
@@ -0,0 +1,252 @@
+"""
```

Comment:
Copied from Pydantic V1. I wasn't sure where to put it so I put it in `schema/_mealie`